libtock/alarm_timer.c

```diff
@@ -4,11 +4,36 @@
 #include <limits.h>
 #include <stdlib.h>
 
+#define MAX_TICKS UINT32_MAX
+
 // Returns 0 if a <= b < c, 1 otherwise
 static int within_range(uint32_t a, uint32_t b, uint32_t c) {
   return (b - a) < (b - c);
 }
 
+inline static uint32_t ticks_to_ms(uint32_t ticks) {
```

Comment:
Does the `inline` keyword actually do anything meaningful on modern compilers? With `static`, this will already be limited to this compilation unit and the compiler can make the inlining decision.